modules/olm-mirroring-catalog.adoc

```diff
@@ -84,7 +84,8 @@ $ oc adm catalog mirror \
     [--filter-by-os="<os>/<arch>"] \//<5>
     [--manifests-only] <6>
 ----
-<1> Specify the index image for the catalog you want to mirror. For example, this might be a pruned index image that you created previously, or one of the source index images for the default catalogs, such as `{index-image-pullspec}`. <2> Specify the target registry to mirror the Operator content to.
+<1> Specify the index image for the catalog you want to mirror. For example, this might be a pruned index image that you created previously, or one of the source index images for the default catalogs, such as `{index-image-pullspec}`.
+<2> Specify the target registry to mirror the Operator content to.
```

Comment:
@adellape Can you check this file too please?

Reply:
Perfect, thanks. A casualty of a recent un-wrap. :grimacing:

Reply:
Yep, unfortunately gotta be careful about that happening!